dao/src/main/java/greencity/dto/user/RegistrationStatisticsDtoResponse.java

```diff
@@ -0,0 +1,14 @@
+package greencity.dto.user;
+
+import lombok.*;
+
+@Getter
+@Setter
+@NoArgsConstructor
```

Comment:
**Comment for everyone.**

One more thing that I noticed, almost in each pull request you are putting almost all Lombok annotations. But here, for example, I would remove `@NoArgsConstructor` and `@Builder` and even `@Setter`